### PR TITLE
WT-8802 Removed duplicate -j argument in linux-directio evergreen task

### PR DESCRIPTION
**Pull request by Sean04**
A second -j 4 argument was added to the linux-directio evergreen task as part of [WT-5458](https://jira.mongodb.org/browse/WT-5458) to increase the number of resources for the test however a patch build shows the default -j (number of processors) to be sufficient.

Evergreen patch build: https://spruce.mongodb.com/version/622028719ccd4e75c3b4b5ee/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC

### CODE REVIEW

**Review by etienneptl — APPROVED** [review]
Thanks for the investigation & cleanup, LGTM!

File: test/evergreen.yml [hunk 1]
```diff
@@ -1915,7 +1915,7 @@ tasks:
       - func: "compile wiredtiger"
       - func: "format test script"
         vars:
-          format_test_script_args: -t 110 -j 4 direct_io=1
+          format_test_script_args: -t 110 direct_io=1
```
Comment (tammybailey) [inline test/evergreen.yml:1918]: I would update the comment above to say something like "This test is run on a higher spec distro due to historical issues with timeout failures. Consider reducing the parallelism if frequent timeouts occur."

How does that sound? In any case, we need to update the comment to remove the `(x4)`.

**Review by tammybailey — APPROVED** [review]
LGTM pending comment update!

File: test/evergreen.yml (hunk 1 above)
Comment (Sean04) [reply test/evergreen.yml:1920]: Yeah, that makes sense, just added it now.

**Pull request merged**